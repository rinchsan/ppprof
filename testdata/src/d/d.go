package d

import (
	"fmt"
	_ "net/http/pprof"
	"runtime"
)

func main() { // want "should set up pprof at the beginning of main"
	runtime.SetBlockProfileRate(1)
	runtime.SetMutexProfileFraction(1)

	fmt.Println("ppprof")
	fmt.Println("ppprof")
}
