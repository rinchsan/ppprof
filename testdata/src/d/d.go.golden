package d

import (
	"fmt"
	_ "net/http/pprof"
	"runtime"
)

func main() {
	runtime.SetBlockProfileRate(1)
	runtime.SetMutexProfileFraction(1)
	go func() {
		log.Fatal(http.ListenAndServe("localhost:6060", nil))
	}()

	// want "should set up pprof at the beginning of main"
	runtime.SetBlockProfileRate(1)
	runtime.SetMutexProfileFraction(1)

	fmt.Println("ppprof")
	fmt.Println("ppprof")
}
